.github/workflows/ci.yml

```diff
@@ -1,55 +1,23 @@
-name: CI
-on:
-  push:
-    branches-ignore:
-      - 'master'
+on: [push, pull_request]
```

Comment:
The execution of the CI was purposely restricted to all branches, except `master`. A change can only go to `master` if it was already checked, because that is a merge requirement. Thus, I didn't see a value of running them on `master` again. Did I miss something?

Reply:
Okay. True. Makes sense, since we are not working with forks, but all on the same repo.